include/deal.II/base/mpi.h

```diff
@@ -1071,6 +1071,25 @@ namespace Utilities
            const T &          object_to_send,
            const unsigned int root_process = 0);
 
+    /**
+     * A generalization of the classic `MPI_Bcast` function that accepts
+     * arbitrary data types `T`, as long as `boost::serialize` accepts `T` as an
+     * argument.
+     *
+     * @param[in] comm MPI communicator.
+     * @param[in] object_to_send An object to send to all processes.
+     * @param[in] root_process The process that sends the object to all
+     * processes. By default the process with rank 0 is the root process.
+     *
+     * @return Every process receives the object sent by the @p root_process
+     */
+
+    template <typename T>
+    T
+    bcast(const MPI_Comm &   comm,
```

Comment:
I would be in favor to name this function without abbreviations, i.e., `broadcast`.
```suggestion
    broadcast(const MPI_Comm &   comm,
```

Reply:
I thought about this name as well so I changed it.

Reply:
I thought about both options when naming the function, so democracy wins!